### DISCUSSION TITLE
How can I get all active sessions

### DISCUSSION THREAD

**Discussion by sk3106** [nextauthjs/next-auth#466] [Help]
**Your question**
How can I get all the active sessions?

**What are you trying to do**
I would like to get all active sessions so I can create an "online users list". I couldn't find anything about this or similar on the internet so it would be great if somebody can help me with this.

**Documentation feedback**
* [ ] Found the documentation helpful
* [x] Found documentation but was incomplete
* [x] Could not find relevant documentation
* [x] Found the example project helpful
* [ ] Did not find the example project helpful

**Comment by SabariVig** [comment]
Is Being Tracked #143

**Comment by ndom91** [comment]
There is no support for this at the moment, but just thinking out loud here - wouldn't it just be a matter of pulling all sessions from the DB that are a) not yet expired and b) unique per user (i.e. only return 1 per user) and you would have a list of all active sessions.

Here's the mysql schema and which fields would need to be queried, etc.: https://next-auth.js.org/schemas/mysql

**Comment by stale** [comment]
Hi there! It looks like this issue hasn't had any activity for a while. It will be closed if no further activity occurs. If you think your issue is still relevant, feel free to comment on it to keep ot open. Thanks!
